libs/Onboarding/src/Onboarding/Accounts/AccountsService.cpp

```diff
@@ -1,19 +1,20 @@
 #include "AccountsService.h"
 
 #include <StatusGo/Accounts/Accounts.h>
+#include <StatusGo/Accounts/AccountsAPI.h>
 #include <StatusGo/General.h>
 #include <StatusGo/Utils.h>
 #include <StatusGo/Messenger/Service.h>
 
-#include <ApplicationCore/Conversions.h>
+#include <Helpers/conversions.h>
 
 #include <optional>
 
 
 std::optional<QString>
 getDataFromFile(const fs::path &path)
 {
-    QFile jsonFile{Status::toString(path)};
+    QFile jsonFile{Status::toQString(path)};
```

Comment:
Just realized, this is Qt6 code? :) No need for the conversion, since Qt6, QFile and other classes accept `std:fs::path` arguments

https://doc.qt.io/qt-6/qfile.html#QFile-5